### ISSUE TITLE
Participation agreement update

### ISSUE THREAD

**Issue by AndrejKiri** [grafana/ux-research#28]

**Comment by josephskim** [comment]
@AndrejKiri Thoughts on reconciling this issue with #22? I propose consolidating our efforts to improve the consent process into a single issue.

**Comment by AndrejKiri** [comment]
> @AndrejKiri Thoughts on reconciling this issue with #22? I propose consolidating our efforts to improve the consent process into a single issue.

@josephskim I would rather keep these two separate because I would like to have participation agreement updates as our Q3 accomplishment. The consent library scope of work wasn't yet defined and at the moment it is not planned for Q3 or Q4. Merging these two together would delay reporting of the participant agreement updates and IMO that would be a pity. What do you think?

**Comment by josephskim** [comment]
@AndrejKiri Good point on the scoping (or the lack thereof) of the consent library work. We shall keep them separate as is then 👍

**Comment by AndrejKiri** [comment]
## Update – Participation Agreement version 1.1
After a [discussion](url) with @josephskim and @anavss, I included incorporated the following changes to the [templates](https://drive.google.com/drive/folders/1wQzObozp8I0ahrVrsm-gDzSjhA_wHBYf):

1. Changing the legal entity name to Raintank
2. Minor formatting changes in the `Information collected` section
3. Adding `Intellectual entity assignment` section
4. Adding a sentence about the possibility of canceling the incentive (solving Amy's situation)

**Comment by AndrejKiri** [comment]
## Update – Feedback from the designers
@anavss reached out to the UX team members that conducted research in the last 6 months to learn about their experience with the Participant Agreement doc and collection process. Following are the takeaways:

1. One didn’t know about the existence of the [Participation Agreement](https://docs.google.com/document/d/1h4aAFaH9d2iatljX4hl3gnrPWtdsWG21I5nxMYYxiP4/edit?usp=sharing) and the need to collect e-signature.
2. Consent is also being collected in Optimal Workshop for card sorting and other unmoderated methods. In this case, the content of the agreement is pasted in the very first question of the screener, and there’s a preset logic that rejects those who select “I do not agree”.
3. There’s an understanding that User Interviews it not to be used to schedule sessions with paying customers, because the session is often scheduled by CSMs. Therefore, there’s no written consent being collected here - only verbal consent in the beginning of the call, before the researcher starts recording it.
4. One suggested to make incentive terms even more clear on the document, stating that “only 1 virtual card will be offered per person, per research project.”
5. One suggested to recycle the team on this topic because there’s a perception of inconsistency in application + there are new team members.
6. When using UI’s flow, it’s unclear to designers where the signed version of the agreement is saved and stored.
